test/sagemaker_tests/pytorch/inference/integration/sagemaker/test_stabilityai.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import absolute_import
+
+import sys
+
+import pytest
+import logging
+
+
+LOGGER = logging.getLogger(__name__)
+LOGGER.setLevel(logging.INFO)
+LOGGER.addHandler(logging.StreamHandler(sys.stdout))
+
+
+@pytest.mark.model("mnist")
+@pytest.mark.processor("gpu")
+@pytest.mark.gpu_test
+@pytest.mark.stabilityai_only
+def test_mnist_distributed_gpu_stabilityai_one(
+    framework_version, ecr_image, instance_type, sagemaker_regions
+):
+    raise Exception("Deliberate error 1")
+
+
+@pytest.mark.model("mnist")
+@pytest.mark.processor("gpu")
+@pytest.mark.gpu_test
+@pytest.mark.skip_stabilityai
+def test_mnist_distributed_gpu_stabilityai_two(
+    framework_version, ecr_image, instance_type, sagemaker_regions
+):
+    raise Exception("Deliberate error 2")
+
+
+@pytest.mark.model("mnist")
+@pytest.mark.processor("gpu")
+@pytest.mark.gpu_test
+def test_mnist_distributed_gpu_stabilityai_three(
```

Comment:
this one needs stability_ai_only marker, it is currently executing on PT tests

Reply:
Yes, I deliberately left it for this commit. Will add the `stability_only` marker in the next commit. Thanks!